scalameta/dialects/shared/src/main/scala/scala/meta/Dialect.scala

```diff
@@ -150,7 +150,9 @@ final class Dialect private (
     // Scala 3 uses proper precedence rules for infix types, unlike Scala 2
     val useInfixTypePrecedence: Boolean,
     // Scala213Source3 and Scala3 allow infix operator being placed after nl
-    val allowInfixOperatorAfterNL: Boolean
+    val allowInfixOperatorAfterNL: Boolean,
+    // Scala 3 will allow dropping braces for block arguments such as `list.map: a =>`
+    val allowFewerBraces: Boolean
```

Comment:
one additional point: currently, "fewer braces" is the name of the experimental import to enable it in the code but at some point it will be unnecessary, and then this name might become confusing. should we make it something like allowLambdaOptionalBraces?

Reply:
I think it should be good enough to keep the information in the comment and it will be easier for people currently to enable the dialect option. In the future, it might only be a problem for contributors, but the comment should suffice.